src/renderer/components/Login.vue

```diff
@@ -1,19 +1,110 @@
 <template>
-  <el-card>
-    <img id="logo" src="~@/assets/logo.svg" alt="Iroha">
-    <span>Login page, Welcome!</span>
+  <el-card class="login-form-container">
+    <img id="logo" class="logo" src="~@/assets/logo.svg" alt="Iroha">
+
+    <el-form ref="form" :model="form" :rules="rules" label-position="top">
+      <el-form-item label="username:" prop="username">
+        <el-input name="username" v-model="form.username"></el-input>
+      </el-form-item>
+
+      <el-form-item label="private key:" prop="privateKey">
+        <el-row type="flex" justify="space-between">
+          <el-col :span="17">
+            <el-input name="privateKey" v-model="form.privateKey"></el-input>
+          </el-col>
+
+          <el-col :span="6">
+            <el-upload
+              action=""
+              :auto-upload="false"
+              :show-file-list="false"
+              :on-change="onFileChosen"
+            >
+              <el-button type="primary">choose</el-button>
+            </el-upload>
+          </el-col>
+        </el-row>
+      </el-form-item>
+
+      <el-form-item label="node ip:" prop="nodeIp">
+        <el-input v-model="form.nodeIp"></el-input>
+      </el-form-item>
+
+      <el-form-item>
+        <el-button type="primary" @click="onSubmit">Login</el-button>
```

Comment:
Let's make this button larger. And maybe center it. Also, adding some space (margin-top) will be good.